.config/tsaoptions.json

```
@@ -0,0 +1,10 @@
+{  
+    "instanceUrl": "https://devdiv.visualstudio.com/",
+    "template": "TFSDEVDIV",
+    "projectName": "DEVDIV",
+    "areaPath": "DevDiv\\NET Fundamentals\\.NET Acquisition\\SDL",
```

```suggestion
    "areaPath": "DevDiv\\NET Fundamentals\\.NET Acquisition\\Source-build-reference-packages",
```

Nikola created this areapath for us already.